ark/internal/genai/memory_http.go

```diff
@@ -127,6 +127,9 @@ func (m *HTTPMemory) AddMessages(ctx context.Context, queryID string, messages [
 		SessionID: m.sessionId,
 		QueryID:   queryID,
 		Messages:  openaiMessages,
+		Ark: ArkMetadata{
```

Comment:
it looks like we might not use `Ark` at all any more is that right? if so it's OK to remove it